app/src/test/java/com/macbitsgoa/ard/models/UserTest.java

```diff
@@ -0,0 +1,33 @@
+package com.macbitsgoa.ard.models;
+
+import com.macbitsgoa.ard.utils.AHC;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+/**
+ * Tests for model User to ensure it's usable to use with firebase db.
+ * @author Rushikesh Jogdand
+ */
+
+public class UserTest {
```

Comment:
Try to use 'set' method to keep coverage high.
otherwise remove them if they are not needed